src/auto_note.cpp

```diff
@@ -330,20 +330,18 @@ void auto_note_manager_gui::show()
     int startPosition = 0;
     int endPosition = 0;
 
-    input_context ctx{ "AUTO_NOTES" };
-    ctx.register_action( "QUIT" );
-    ctx.register_action( "SWITCH_AUTO_NOTE_OPTION" );
-    ctx.register_action( "HELP_KEYBINDINGS" );
-    ctx.register_action( "NEXT_TAB" );
+    input_context ctxt{ "AUTO_NOTES" };
+    ctxt.register_action( "QUIT" );
+    ctxt.register_action( "SWITCH_AUTO_NOTE_OPTION" );
+    ctxt.register_action( "HELP_KEYBINDINGS" );
+    ctxt.register_action( "NEXT_TAB" );
     if( !char_emptyMode || !global_emptyMode ) {
-        ctx.register_cardinal();
-        ctx.register_action( "PAGE_UP", to_translation( "Fast scroll up" ) );
-        ctx.register_action( "PAGE_DOWN", to_translation( "Fast scroll down" ) );
-        ctx.register_action( "CONFIRM" );
-        ctx.register_action( "QUIT" );
-        ctx.register_action( "ENABLE_MAPEXTRA_NOTE" );
-        ctx.register_action( "DISABLE_MAPEXTRA_NOTE" );
-        ctx.register_action( "CHANGE_MAPEXTRA_CHARACTER" );
+        ctxt.register_navigate_ui_list();
+        ctxt.register_action( "CONFIRM" );
+        ctxt.register_action( "QUIT" );
+        ctxt.register_action( "ENABLE_MAPEXTRA_NOTE" );
+        ctxt.register_action( "DISABLE_MAPEXTRA_NOTE" );
+        ctxt.register_action( "CHANGE_MAPEXTRA_CHARACTER" );
```

Comment:
This rename is for consistency with other UI source files, the vast majority of which use this consistent name.